.github/workflows/release.yml

```diff
@@ -8,12 +8,31 @@ on:
   push:
     branches:
       - main
-      - master
     paths:
-      - .bumpversion.cfg
+      - Dockerfile.tools
```

Comment:
This implies that the releases will be based on this file. However, this file includes both packer and terraform. And `make release/version` is based on only the terraform version. So won't this become a bit of an issue with the packer version gets updated, since this release workflow would be triggered?

Reply:
Maybe not. The workflow will run, but the release condition won't be met, so it would stop before attempting an actual release.

Reply:
Maybe we want more continual releases for this project? Any change to any of the files that track packages we mirror, we release. Instead of a semver scheme, the release could match the git sha?

Reply:
It would be more accurate with updates to the project.  Are there any limits on releases with GitHub? If not, I'm okay with continual releases.

Reply:
I don't _know_ of limits? Things could get weird if many updates are being processed simultaneously. But seems worth a try...

Reply:
I switched to using the git sea as the release.  Hope that approach works.